lib/node_modules/@stdlib/math/base/special/besselj0/README.md

```diff
@@ -0,0 +1,62 @@
+# besselj0
```

Comment:
For the README and function alias within the package, you can just use `j0`. The package name is fine. In the REPL, we can alias as `besselj0`.